recipes/mender-configure/debian-master/rules

```diff
@@ -4,7 +4,7 @@
 	dh $@
 
 override_dh_auto_install:
-	DESTDIR=debian/mender-configure make install
+	DESTDIR=debian/tmp make install install-scripts install-demo
```

Comment:
Oh, so is this "the right way" of doing it? Are the files picked up from there and into their respectively packages folders?

Reply:
Yes, it's a bit strange, but `tmp` is actually documented [here](https://www.debian.org/doc/manuals/maint-guide/dother.en.html#install), as the place to install things. And then the builder will pick it up from there.

Reply:
Thanks